src/coredump/coredump.c

```diff
@@ -668,8 +668,7 @@ static int get_process_ns(pid_t pid, const char *namespace, ino_t *ns) {
         return 0;
 }
 
-static int get_mount_namespace_leader(pid_t pid, pid_t *container_pid) {
```

Comment:
nit: might be 'cleaner' to have the coredump changes in a separate commit

Reply:
must be in the same, since this just follows the API change. i.e. if it would be in two, then get_process_ppid() would return EADDRNOTAVAIL but coredump.c would propagate that.

I mean, it's no biggie, but we have this rule that commits should not fix up stuff commits in the *same* PR break. But if we'd split this up then we'd have a "broken" state in the middle, were coredump.c wouldn#t be able to deal with the new errors.

Reply:
fair enough